share/getters/shrex.go

```diff
@@ -18,23 +18,28 @@ import (
 
 var _ share.Getter = (*ShrexGetter)(nil)
 
-const defaultMaxRequestDuration = time.Second * 10
+const (
+	defaultMinRequestTimeout = time.Second * 10
```

Comment:
shouldn't this be based in min attempts count ?

Reply:
No, it is absolute minimal timeout to try sample from peer. We need to limit lowest edge, because it is reasonable time to sample large block from healthy peer.  Timeout could not be less than this value. Real timeout could be higher and affected by sampling timeout configuration (workers amount*blocktime) / minAttempts.